build.gradle

```diff
@@ -3540,6 +3540,7 @@ project(':connect:runtime') {
     // We use this library to generate OpenAPI docs for the REST API, but we don't want or need it at compile
     // or run time. So, we add it to a separate configuration, which we use later on during docs generation
     swagger libs.swaggerJaxrs2
```

Comment:
@cshannon you are right - it is better to use `swagger-jaxrs2-jakarta` instead. We can use following dependencies to resolve the issue:
```
    swagger 'jakarta.servlet:jakarta.servlet-api:6.1.0'
    swagger 'io.swagger.core.v3:swagger-jaxrs2-jakarta:2.2.25'
```

Reply:
> @cshannon you are right - it is better to use `swagger-jaxrs2-jakarta` instead. We can use following dependencies to resolve the issue:
> 
> ```
> 
>     swagger 'jakarta.servlet:jakarta.servlet-api:6.1.0'
> 
>     swagger 'io.swagger.core.v3:swagger-jaxrs2-jakarta:2.2.25'
> 
> ```

That's good news, sorry for missing that on the original PR, I didn't realize it was used for the swagger generation.

Reply:
Thank for @chia7712, @cshannon suggestion, addressed it

Reply:
> That's good news, sorry for missing that on the original PR, I didn't realize it was used for the swagger generation.

That's completely fine. Thank you for helping us resolve the issue! :)